src/GraphQL/Validation/Complexity/ComplexityResult.cs

```diff
@@ -3,10 +3,24 @@
 
 namespace GraphQL.Validation.Complexity
 {
+    /// <summary>
+    /// Contains the result of a complexity analysis.
+    /// </summary>
     public class ComplexityResult
     {
+        /// <summary>
+        /// Returns a dictionary of nodes and their complexity factors.
+        /// </summary>
         public Dictionary<INode, double> ComplexityMap { get; } = new Dictionary<INode, double>();
+
+        /// <summary>
+        /// Returns the calculated document complexity factor.
+        /// </summary>
         public double Complexity { get; set; }
+
+        /// <summary>
+        /// Returns the total query depth.
+        /// </summary>
         public int TotalQueryDepth { get; set; }
```

Comment:
To be honest, I don't understand the meaning of `Total` here.

Reply:
Judging by the code, this is the _actual_ maximum depth of the executed request.

Reply:
I suggest to rename `Complexity` into `TotalComplexity` and `TotalQueryDepth` to `ActualQueryDepth` into separate PR. What do ypu think?

Reply:
I suggest `TotalComplexity` and `QueryDepth`.  Whatever you want.  I do want to use this for my site, but have not done so yet.  Most likely I will write my own implementation anyway, since the most important point for me will be assigning a cost factor to each node, which I don't think is possible in the current implementation.  (Maybe it is; I have not looked.)

Reply:
> I suggest TotalComplexity and QueryDepth

👍